README.md

```diff
@@ -1,109 +1,11 @@
-# Great Expectations Pipeline Tutorial
+# Great Expectations tutorials
```

Comment:
Most of the content here was moved into the README in the ge_dbt_airflow_tutorial subdirectory.